### DISCUSSION TITLE
complejidades fuera de eventos

### DISCUSSION THREAD

**Discussion by GuilleStankeS** [IIC2133-PUC/2024-2#89] [Tarea 1] [answered]
Hola... en el enunciado se pide no superar distintas complejidades en cada evento, sin embargo... en la creación de estructuras o código inicial que se ejecuta solo una vez en un comienzo (antes de recibir y trabajar el primer evento), existe una complejidad de tiempo máxima permitida? o puedo tener un algoritmo con complejidad O(n) por ejemplo en el comienzo??

**Comment by sebacarrascop** [comment, marked as answer]
hola, solo se piden complejidades de los eventos y cumplir con los tiempos.
